app/decorators/swagger/project_decorator.rb

```diff
@@ -2,21 +2,29 @@ module Swagger
   class ProjectDecorator < Draper::Decorator
     delegate_all
 
-    def to_swagger
+    def to_swagger(with_api_gateway_integration = false)
       {
         openapi: '3.0.0',
         info: {
           description: description,
-          title: title,
+          title: info_title(with_api_gateway_integration),
           version: '1.0.0'
         },
         servers: servers,
         tags: tags,
-        paths: paths,
+        paths: paths(with_api_gateway_integration),
```

Comment:
You can choose using keyword parameter or not. As you are in private methods it may be fine for you to keep `paths(with_api_gateway_integration)` rather than heavier `paths(with_api_gateway_integration: with_api_gateway_integration)`

Reply:
Hmm… Is it just a complement to the comment https://github.com/applidium/pericles/pull/6#discussion_r330417388 or is there another issue?

Reply:
It's just a matter of coherence. As you change the signature of the main (public) method, you may want to change the signature of the internal (private) method. But as I said it's up to you.